inc/Engine/Optimization/RUCSS/Database/Queries/ResourcesQuery.php

```diff
@@ -78,6 +78,8 @@ public function create_or_update( array $resource ) {
 		// check the database if those resources added before.
 		$db_row = $this->get_item_by( 'url', $resource['url'] );
 
+		$resource['content'] = addslashes( $resource['content'] );
```

Comment:
This should happen in Resource and not in Used CSS?
Isn't this the error coming from Used CSS or I am missing something?

Reply:
No, the issue was at warmup, when trying to save the resource content into the DB.
But your comment gets into my mind that we need to apply the same at the used_css table also while saving it.

Reply:
Please test it round up: Warmup => Saas Warmup => TreeShake => Used CSS to make sure that is 100% covered .
I am kinda 90% sure this will happen in Used CSS DB table too.